gwcli/ceph.py

```diff
@@ -288,11 +288,18 @@ class RadosPool(UINode):
     def __init__(self, parent, pool_name, pool_md):
         UINode.__init__(self, pool_name, parent)
         self.pool_md = pool_md
+        k = 'N'
+        m = 'N'
+        if pool_md['type'] == 3:
+            result, err = run_shell_cmd(
+                "ceph osd erasure-code-profile get {profile} --format json".
```

Comment:
Nit: no need to name the argument here, use `{}` -- it's shorter ;)

Reply:
Will fix it.